sysml.library/Kernel Library/VectorFunctions.kerml

```diff
@@ -0,0 +1,192 @@
+/**
+ * This package defines abstraft functions on VectorValues corresponding to the algebraic operations
```

Comment:
typo: abstract

Reply:
Fixed in commit [3505da2](https://github.com/Systems-Modeling/SysML-v2-Pilot-Implementation/pull/337/commits/3505da240c3983885d275d65d636fde6adcf7ad6).